nautobot/extras/templates/extras/job_bulk_edit.html

```diff
@@ -0,0 +1,75 @@
+{% extends 'base.html' %}
```

Comment:
Not `generic/object_bulk_update.html`?

Don't hesitate to add a new `{% block %}` in that template if need be, to allow you to override just the part of it that's needed.